src/dftbp/dftbplus/parser.F90

```diff
@@ -1564,7 +1563,7 @@ subroutine readDFTBHam(node, ctrl, geo, slako, poisson)
   #:endif
 
     ! K-Points
-    call readKPoints(node, ctrl, geo, tBadIntegratingKPoints)
+    call readKPoints(node, ctrl, geo, ctrl%areBadIntegratingKPoints)
```

Comment:
Wouldn't be `poorKSampling` shorter? (I would suggest to avoid variable names over 20 chars, if shorter synonyms can be found)

Reply:
Sure (my own fault for trying to enforce the t convention for naming without thinking about it much)